README.md

```diff
@@ -25,7 +25,7 @@ community connectors.
 1. Clone the repo from https://github.com/DataDog/datadog-kafka-connect-logs
 2. Verify that Java8 JRE or JDK is installed.
 3. Run `mvn clean compile package`. This will build the jar in the `/target` directory. The name will be `datadog-kafka-connect-logs-[VERSION].jar`.
```

Comment:
What OS are you building on? It seems like the confluent `.jar` will not build on Windows: https://stackoverflow.com/questions/60993427/kakfa-schema-registry-maven-build-failure-kafka-connect

Reply:
Otherwise, what is your JDK version? Can you clean your local maven cache?

```
Unix/Mac OS X – ~/.m2/repository
Windows – C:\Documents and Settings\{your-username}\.m2\repository 
```

Reply:
It works with Java8